src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    private List<Excavator> excavators = new ArrayList<>();
```

Comment:
Don't forget about access modifiers for your custom fields. The 'excavators' list should be private to prevent direct access from outside of the class.